alertwildfire_live_recording.py

```diff
@@ -97,13 +99,22 @@ def capture_and_record(googleServices, outputDir, camera_name):
             retriesLeft -= 1
 
     image_base_name = pathlib.PurePath(imgPath).name
-    image_name_with_metadata = build_name_with_metadata(image_base_name,metadata)
+    image_name_with_metadata = build_name_with_metadata(image_base_name,pull1)
     cloud_file_path =  'alert_archive/' + camera_name + '/' + image_name_with_metadata
     goog_helper.uploadBucketObject(googleServices["storage"], settings.archive_storage_bucket, cloud_file_path, imgPath)
-
     
 
 
+    dbManager  = db_manager.DbManager(psqlHost = settings.psqlHost,
+ psqlDb = settings.psqlDb,
+ psqlUser = settings.psqlUser,
+ psqlPasswd = settings.psqlPasswd)
```

Comment:
dbManager should be instantiated once at the beginning of the process.  and passed in as a parameter to this function.  Even better would be to use the code in detect_fire.py that selects between sqlite and postgress at dbManager initialization.  Would be best to move that selection code into db_manager.py to avoid code duplication.